navit/navit_layout_car_shipped.xml

```diff
@@ -175,11 +175,11 @@
 			<text text_size="5"/>
 		</itemgra>
 		<itemgra item_types="poly_construction" order="10-">
-			<polygon color="#f47216"/>
+			<polygon color="#cc776699"/>
```

Comment:
OK, it doesn't hurt to have transparency here as poly_construction is used quite seldom. And I can imagine a lot of mapping errors where poly_construction overlaps something it shouldn't according to OSM.